tools/dep_updaters/update-postject.sh

```diff
@@ -7,6 +7,15 @@
 
 set -ex
 
+NEW_VERSION=$(npm view postject dist-tags.latest)
+CURRENT_VERSION=$(node -p "require('./test/fixtures/postject-copy/node_modules/postject/package.json').version")
```

Comment:
We wouldn't want to use the values on the path in these scripts, for consistency.
```suggestion
NEW_VERSION=$("$NODE" "$NPM" view postject dist-tags.latest)
CURRENT_VERSION=$("$NODE" -p "require('./test/fixtures/postject-copy/node_modules/postject/package.json').version")
```

Reply:
I 100% agree! Also, I'd like to do it in others scripts too ^^